framework/resources/Functional/filter/pushdown/plan/DRILL_6174_filter_push_down_not_is_null_plan.res

```diff
@@ -0,0 +1,5 @@
+.*Scan
```

Comment:
Doesn't look like valid regex to me, as the tokens are on different lines. Are you sure it works? I tried a sample string using an online tool such as https://regex101.com/ and the regex did not match. 

I would expect these to be in the same lines, separated by .*, for example, 
.*Scan.*path=blah.*numFiles.* 
As we want to match the text portion of the plan with the regex - which is usually one line per operator (here Scan).

Reply:
It appears that Drill Test Framework trims whitespaces and new line characters from the result file.  I was guided by the example of some other tests.
At least, it actually works and tests pass.

Reply:
OK, fixed.